### PR TITLE
Link BlankApp project with TestFixtures.

### PR DESCRIPTION
**Pull request by jasssonpet**
@fealebenpae Any better way to do this?

### CODE REVIEW

**Comment by fealebenpae** [comment]
How about we create a new macro, say `LinkWithTestFixtures`, that accepts a target, possibly created by `CreateNativeScriptApp`, and adds the necessary dependencies and link flags?

**Comment by jasssonpet** [comment]
Shouldn't the include directories be set before the target is created.

**Comment by fealebenpae** [comment]
[`target_include_directories`](http://www.cmake.org/cmake/help/v3.0/command/target_include_directories.html)?

**Comment by jasssonpet** [comment]
That's just what I was looking for, thanks :)

**Comment by jasssonpet** [comment]
@fealebenpae This PR is working on my machine. We should fix the PR build to reconfigure WebKit as needed.

**Comment by jasssonpet** [comment]
run ci

File: cmake/LinkTestFixtures.cmake [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function(LinkTestFixtures _target)
+    set(TESTFIXTURES_DIR "${CMAKE_SOURCE_DIR}/tests/TestFixtures")
+
+    file(READ "${TESTFIXTURES_DIR}/exported-symbols.txt" TESTFIXTURES_EXPORTED_SYMBOLS)
```
Comment (fealebenpae) [inline cmake/LinkTestFixtures.cmake]: I did a little poking around, and it appears `file(STRINGS "${TESTFIXTURES_DIR}/exported-symbols.txt" TESTFIXTURES_EXPORTED_SYMBOLS)` can read the contents of a the exported symbols list file and split its lines into a list. Can we use it instead?

Comment (jasssonpet) [inline cmake/LinkTestFixtures.cmake]: Done

**Pull request merged**